src/utils/api.js

```diff
@@ -4,9 +4,23 @@ const api = axios.create({
     baseURL: 'https://news-app-2.onrender.com/api',
   });
 
-export const getArticles = ()=>{
+export const getArticles = (topic, query)=>{
+    let queryStr = ''
+
```

Comment:
Great use of conditions to create a suitable query string, however you could use axios' params argument to remove the need for a lot of this logic https://notes.northcoders.com/courses/js-front-end/axios#axios-params